src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -0,0 +1,35 @@
+from sentry.integrations.slack.utils import LEVEL_TO_COLOR
+from sentry.utils.assets import get_asset_url
+from sentry.utils.http import absolute_uri
+
+
+def build_notification_attachment(notification, context):
+    logo_url = absolute_uri(get_asset_url("sentry", "images/sentry-email-avatar.png"))
+    settings_url = absolute_uri("/settings/account/notifications/")
```

Comment:
We could make this more specific - like send the user to the fine tuning page depending on the type of notification.